assets/js/vue/components/research-group/PeopleTab.vue

```diff
@@ -43,11 +40,25 @@
         column-gap: 1rem !important;
         column-count: 2 !important;
         .card {
-            height: 200px;
+            height: 12em;
         }
     }
+    .card-product {
+        margin-bottom: 1rem;
```

Comment:
```suggestion
        margin-bottom: 1rem;
        transition: .5s;
```
More a more natural transition, also add the transition out.